### DISCUSSION TITLE
Is there no need to handle HAL_GetTick overflows

### DISCUSSION THREAD

**Discussion by wizhippo** [Ultrawipf/OpenFFBoard#101] [General]
Is the startTime always less then the overflow?

If not these should be adjusted to account for the tick counter rollover

https://github.com/Ultrawipf/OpenFFBoard/blob/dc2d3793e4d1d515c6ef6987722c126f736c2f8d/Firmware/FFBoard/Src/EffectsCalculator.cpp#L121

https://github.com/Ultrawipf/OpenFFBoard/blob/dc2d3793e4d1d515c6ef6987722c126f736c2f8d/Firmware/FFBoard/Src/EffectsCalculator.cpp#L125

**Comment by Ultrawipf** [comment]
You are right overflows should be handled and i will change these sections.
But as it is so rare that an effect is started with delay at exactly the time the 32b 1ms tick overflows every 50 days its unlikely to have any impact.
